tests/alternative/GPM.lua

```diff
@@ -31,19 +29,6 @@ return {
 		local error_func = function()
 			GPM{
 				destination = network,
-				origin = farms
-			}
-		end
-		unitTest:assertError(error_func, "The CellularSpace in argument 'origin' must be loaded with 'geometry = true'.")
```

Comment:
All these alternative tests should be executed. The error should occur now when someone creates a `CellularSpace` using `geometry = false`.

Reply:
I think this is no test for GPM, it is test for CS. GPM must test its functionalities. What do you think?

Reply:
If someone uses a `CellularSpace` with `geometry = false` as argument to GPM, it cannot read the geometries, and therefore it must stop with an error.

Reply:
Actually, there are codes associated to that. I going to review.